profile.md

```diff
@@ -1447,10 +1447,10 @@ In this example, the `nonce`, `preferred_username`, `name`, and `email` claims a
      From GFD-I.182, the VOMS spec for FQANs (Sect. 3.4.1.4)
 
 [^6]:
-     https://aarc-project.eu/guidelines
+     [AARC guidelines](https://aarc-project.eu/guidelines)
```

Comment:
We probably should refer directly to [AARC-G069](https://aarc-community.org/guidelines/aarc-g069/)

Reply:
Updated via [this](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/commit/d092832e9efe09245e0c3309c9c1601d943ec7db) commit.